tokio/src/lib.rs

```diff
@@ -393,6 +393,7 @@
 //!  * Android (API level 21)
 //!  * macOS
 //!  * iOS
+//!  * tvOS
```

Comment:
We're not going to add a tier 3 target to this list.

Reply:
Rereading it I realize now I shouldn't have touched that, sorry about that.

> Thanks. Would you be able to add a cargo check for this target to CI?

Sure, I think so, will add.

Reply:
You can check my last commit where I add the CI job, however this target requires ["appropriate versions of Xcode and the SDKs for tvOS (AppleTVOS.sdk) and/or the tvOS Simulator (AppleTVSimulator.sdk) to build a toolchain and target these platforms."](https://doc.rust-lang.org/nightly/rustc/platform-support/apple-tvos.html)

Reply:
Yes, so far so good! :)